cropper/src/main/java/com/canhub/cropper/BitmapCroppingWorkerJob.kt

```diff
@@ -147,7 +147,7 @@ class BitmapCroppingWorkerJob internal constructor(
                             flipVertically
                         )
                     } else {
-                        onPostExecute(Result(null as Bitmap, 1))
+                        onPostExecute(Result(bitmap = null, 1))

```

Comment:
I am using named parameters
Since I am calling that function in Kotlin file, I can take advantage of named parameters..

Previously casting null as Bitmap showing a warning `This cast can never succeed`.
: )